linux_os/guide/services/ntp/chronyd_specify_remote_server/oval/shared.xml

```diff
@@ -15,7 +15,7 @@
 
   <ind:textfilecontent54_object comment="Ensure at least one NTP server is set"
   id="object_chronyd_remote_server" version="1">
-    <ind:filepath operation="pattern match">^/etc/chrony\.(conf|d/.+\.conf)$</ind:filepath>
+    <ind:filepath operation="pattern match">^/etc/(?:chrony/)?chrony\.(conf|d/.+\.conf)$</ind:filepath>
```

Comment:
This needs to be reworked to use product properties `chrony_conf_path` and `chrony_d_path` correctly.

Reply:
commit has been amended accordingly.